tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,92 @@
+/* globals fluid, jqUnit */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+
+    // Component to test support for common event bindings
+    fluid.defaults("gpii.tests.binder.commonEventBindings", {
+        gradeNames: ["gpii.tests.binder.base", "gpii.binder.bindMarkupEvents"],
+        bindings: {
+        },
+        selectors: {
+            inputButton: "#input-button",
+            paragraphClick: ".parapgraph-click"
+        },
+        markupEventBindings: {
+            inputButton: {
```

Comment:
You really need to add examples that include multiple bindings for a single element.  For example, I often want to add `click` and `onKeyDown` to elements so that keyboard and mouse navigation are both supported.

Reply:
Interesting, it didn't actually have support for this, but it makes total sense. Added impl and tests to make this possible (without making duplicate selectors)

Reply:
Great, I had my own one-offs for this and it'll be nice to have an alternative.